Dockerfile

```
@@ -0,0 +1,11 @@
+FROM node:18-slim
+
+WORKDIR /usr/src/app
+COPY package*.json .
+RUN npm install
+COPY . .
+RUN npm run build
+RUN npm install -g .
+RUN chmod +x entrypoint.sh
```

I think a simple Dockerfile like the following would work fine.

```suggestion
COPY . /app
WORKDIR /app
RUN npm ci
RUN npm run build
RUN npm install -g .
RUN chmod +x entrypoint.sh
```